### PR TITLE
Add farmlands card partner support guide

### PR DESCRIPTION
**Pull request by laurencemcknight**
Adds the documentation from:
https://www.notion.so/centrapay/Farmlands-Card-Partner-Support-Guide-eaae9d64d022408aa666ea1936f9c5ae?pvs=4

I have shown this page running locally to Neza and Paul and they have given their approval.

### CODE REVIEW

File: src/content/guides/farmlands-card-partner-support.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Farmlands Card Partner Support Guide
+img: /farmlands-pos-integration-cover.jpg
+description: How to for Farmlands Card Partners and Card Holders to request support from the relevant company and teams to get the help they need.
```
Comment (tom-blake) [inline src/content/guides/farmlands-card-partner-support.mdx:4]: I think there's an opportunity to make this shorter. Longer SEO descriptions will be truncated on some sites so it's nice to make as much visible to the reader as possible. Ultimately, we want them to click the link 💪 

```suggestion
description: How to for Farmlands Card Partners and Card Holders to get the help they need.
```

File: src/content/guides/farmlands-card-partner-support.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+title: Farmlands Card Partner Support Guide
+img: /farmlands-pos-integration-cover.jpg
```
Comment (tom-blake) [inline src/content/guides/farmlands-card-partner-support.mdx:3]: Do we want to use a different image? This image was for the POS guide

File: src/content/guides/farmlands-card-partner-support.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  path: Connections/Farmlands
+  title: Support Guide
+  order: 2
+---
+
+If you are a Farmlands Card Partner needing extra support this is the guide for you. Here you will find who you need to reach out to and how you can get in touch with them.
+
+## Support for Portal Users
+
+1. If you require assistance with Portal functionality, contact Centrapay at support@centrapay.com or Farmlands at card.specialist@farmlands.co.nz
```
Comment (tom-blake) [inline src/content/guides/farmlands-card-partner-support.mdx:15]: Do these email links render as `mailto:` links?

`mailto:` links are a nice UX touch

File: src/content/guides/farmlands-card-partner-support.mdx (hunk 2 above)
Comment (laurencemcknight) [reply src/content/guides/farmlands-card-partner-support.mdx:3]: We can, there is not currently a specific image that we are supposed to use for this guide so I reused the POS guide image. I will contact the Jesters to see if they have something in mind.

File: src/content/guides/farmlands-card-partner-support.mdx (hunk 3 above)
Comment (laurencemcknight) [reply src/content/guides/farmlands-card-partner-support.mdx:15]: I will add those in now

Comment (laurencemcknight) [reply src/content/guides/farmlands-card-partner-support.mdx:15]: Good news! Astro automatically recognises that those are email addresses and opens the mail client when clicked. Therefore the mailto tag is unnecessary.

Comment (tom-blake) [reply src/content/guides/farmlands-card-partner-support.mdx:15]: That's cool 😎

File: src/content/guides/farmlands-card-partner-support.mdx (hunk 2 above)
Comment (tom-blake) [reply src/content/guides/farmlands-card-partner-support.mdx:3]: @laurencemcknight Sweet - I'd suggest removing this line entirely if they don't have one ready to go.

We default to the image that you see on our core guides if there's no `img` property in the front matter. Chat with them to see which is the most appropriate option 👍 

I'll approve this PR to unblock it

**Review by tom-blake — APPROVED** [review]

**Review by IsaacMurtagh — APPROVED** [review]

File: src/content/guides/farmlands-card-partner-support.mdx (hunk 2 above)
Comment (laurencemcknight) [reply src/content/guides/farmlands-card-partner-support.mdx]: Jesters have said to use the default background image here. I will remove the POS integration cover image.

**Pull request merged**